style.css

```diff
@@ -0,0 +1,386 @@
+* {
+	box-sizing: border-box;
+	margin: 0;
+	font-size: 10px;
+	font-family: Ubuntu, Sans-serif;
+	color: white;
+	font-weight: bold;
+}
+br {
+	display:none;
+}
+body {
+	background-color: #C06C84 ;
+	width: 100%;
+	font-family: "Idim";
+}
+nav {
+	width: 100%;
+	display: flex;
+	justify-content: space-between;
+	height: 50px;
+	position: fixed;
+	padding: 15px;
+	background-color: #C06C84;
+}
+
+footer {
+	text-align: center;
+	background-color: #355C7D;
+	padding-bottom: 30px;
+}
+
+a:visited {
+	color: inherit;
+}
+
+
+
+.nav-left-container {
+	display: flex;
+	justify-content: flex-start;
+	gap: 30px;
+	color: white;
+}
+
+.nav-left-container a {
+	text-decoration: none;
+	font-size: 2rem;
+	color: white;
+}
+
+nav > a {
+	font-size: 2rem;
+	text-decoration:none;
+}
+
+.intro {
+	display: flex;
+	justify-content: center;
+	flex-wrap: wrap-reverse;
+	padding: 15px;
+	align-items: center;
+	max-width: 1000px;
+	margin: 0 auto;
+	height: 100%;
+	gap: 30px;
+	
+
+	
+}
+
+.intro > img {
+	width: 300px;
+	object-fit: cover;
+	border-radius: 5px;
+}
+.intro-texts > h2 {
+	font-size: 3.5rem;
+	letter-spacing: 3px;
+	line-height: 80px;
+}
+.intro-texts > h1 {
+	font-size: 7rem;
+	letter-spacing: 1.5px;
+	width: 100%;
+	line-height: 80px;
+	margin: 35px 0;
+}
+
+.intro-texts > h3 {
+	font-size: 2.4rem;
+	letter-spacing: 2px;
+	line-height: 40px;
+}
+
+
+
+
+.intro-texts{
+	max-width: 65%
+}
+
+.about-me {
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-left;
+	flex-wrap: wrap;
+	gap: 50px;
+	align-items: center;
+	max-width: 1000px;
+	width: 100%;
+	margin: 0 auto;
+	background-color: #355C7D;
+	padding: 15px;
+	
+}
+.about {
+	background-color:#355C7D;
+	border: 50px solid #355C7D ;
+	width: 100%;
+}
+
+.about-me-text{
+	max-width:70%;
+
+}
+
+.about-me-text > h2, .portfolio-title-container > h2 {
+	font-size: 3.5rem;
+	letter-spacing: 3px;
+	margin-bottom: 15px;
+}
+
+.about-me-text > p, .portfolio-title-container > h3{
+	font-size: 2rem;
+	width: 100%;
+	letter-spacing: 2px;
+	line-height: 30px;
+}
+
+.full-stack-track-link {
+	font-size: 2rem;
+	text-decoration: none;
+	color: #C06C84;
+}
+
+.full-stack-track-link:visited {
+	color: #C06C84;
+}
+
+.about-me > img {
+	width:20%;
+	border-radius: 3px;
+}
+
+.portfolio {
+	display: flex;
+	justify-content: center;
+	flex-direction: column;
+	align-items: center;
+	width: 100%;
+	margin: 0 auto;
+	border-top: 50px solid  #355C7D ;
+	border-bottom: 50px solid  #355C7D ;
+	background-color: #a1596e;
+
+}
+
+.portfolio-title-container {
+	max-width: 1000px;
+	width: 100%;
```

Comment:
there are a few elements that have both these rules. you should write them all in one place so that e.g. if you changed the max-width to 900px, you only need to change it in one place and they would still match

Reply:
done!